### ISSUE TITLE
Resolve duplicate content on API interfaces

### ISSUE THREAD

**Issue by seanmadsen** [civicrm/civicrm-dev-docs#436]
Duplicate content here: 

* REST
    * https://docs.civicrm.org/dev/en/latest/api/usage/#rest
    * https://docs.civicrm.org/dev/en/latest/api/interfaces/#rest
* AJAX
    * https://docs.civicrm.org/dev/en/latest/api/interfaces/#ajax
    * https://docs.civicrm.org/dev/en/latest/api/usage/#ajax

And other sections in those two pages "smarty" etc...

I think the "Interfaces" page is probably the better place for this content. We should move content out of the "Usage" page and into the "Interfaces" page in order to resolve this duplication.

**Comment by totten** [comment]
* Overall, I like the structure, intro, and navigational placement of "API Usage" better. (Although I don't like the prose that's under REST/AJAX/Smarty.)
 * The fine-grained details of "API Interfaces" would make more sense as separate pages.
 * The last two things in "API Usage" seem like random after-thoughts -- like someone couldn't think of a place to those notes, so they plunked it down on the "Usage" page.

**Comment by seamuslee001** [comment]
I agree with Tim's points, I think it makes sense to go under usage. I think the creating an API key probably should go somewhere in the introduction or similar. Separate pages makes good amount of sense. Usage is an overview with more detail on how to use them on their individual pages
